src/Symfony/Component/Security/Http/RememberMe/RememberMeDetails.php

```diff
@@ -21,14 +21,20 @@ class RememberMeDetails
 {
     public const COOKIE_DELIMITER = ':';
 
-    private $userFqcn;
     private $userIdentifier;
     private $expires;
     private $value;
 
-    public function __construct(string $userFqcn, string $userIdentifier, int $expires, string $value)
+    public function __construct(/*string*/ $userIdentifier, /*int*/ $expires, /*string*/ $value)
```

Comment:
you need to add the phpdoc to declare the method signature (for IDEs, static analyzers, etc...)